ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java

```diff
@@ -139,33 +111,35 @@ public void onResult(String consumerSessionId, Exception error) {
     }
 
     /**
-     * Continues the 3DS verification. Should be called from {@link ThreeDSecureLookupCallback#onResult(ThreeDSecureRequest, ThreeDSecureLookup, Exception)}
-     * @param activity           Android FragmentActivity
-     * @param request            the {@link ThreeDSecureRequest} with information used for authentication.
-     * @param threeDSecureLookup the {@link ThreeDSecureLookup} returned for this request.
- *                           Contains information about the 3DS verification request that will
- *                           be invoked in this method.
-     * @param callback           {@link ThreeDSecureResultCallback}
+     * Continues the 3DS verification. Should be called from {@link ThreeDSecureResultCallback#onResult(ThreeDSecureResult, Exception)}
+     *
+     * @param activity Android FragmentActivity
+     * @param request  the {@link ThreeDSecureRequest} with information used for authentication.
+     * @param result   the {@link ThreeDSecureResult} returned for this request.
+     *                 Contains information about the 3DS verification request that will
+     *                 be invoked in this method.
+     * @param callback {@link ThreeDSecureResultCallback}
      */
-    public void continuePerformVerification(final FragmentActivity activity, final ThreeDSecureRequest request, final ThreeDSecureLookup threeDSecureLookup, final ThreeDSecureResultCallback callback) {
+    public void initiateChallengeWithLookup(final FragmentActivity activity, final ThreeDSecureRequest request, final ThreeDSecureResult result, final ThreeDSecureResultCallback callback) {
```

Comment:
Nitpick - on iOS we call this `initializeChallengeWithLookup`. Maybe we should update this to match?

Reply:
Actually, I think I'm getting confused. On iOS, there's a method called `initializeChallengeWithLookup` that's used by merchants who do server-side 3DS lookups.

This looks like it's used to continue the regular 3DS flow after inspecting the lookup response. Is that right?

I guess I want to understand the name changes better. `performVerification` and `continuePeformVerification` seem to go together nicely. Is there a reason why we're changing them?

Reply:
This one was a bit weird for us too. We had `performVerification` and `continuePerformVerificaton` before, but we didn't see an equivalent in iOS. We went digging to see if we could find something comparable and came up with these terms. We realize that everything on iOS goes through a PaymentFlowDriver extension that has a `startPaymentFlow` method. We'd like to have some form of parity with the naming, but we'd be open to reverting to `performVerification` and `continuePerformVerification`.

Reply:
Yeah, it's a bit different on iOS because we use `BTPaymentFlowDriver`. So `performVerification` is equivalent to `startPaymentFlow` (with a `BTThreeDSecureRequest` being passed in`).

Merchants also have to implement `BTThreeDSecureRequestDelegate`, which has the [`onLookupComplete`](https://braintree.github.io/braintree_ios/current/Protocols/BTThreeDSecureRequestDelegate.html#/c:objc(pl)BTThreeDSecureRequestDelegate(im)onLookupComplete:lookupResult:next:) method. One of the params that we pass to the merchant in the `onLookupComplete` method is a function called `next`. The merchant calls this when they're ready to continue the 3DS flow, which is the equivalent of calling `continuePerformVerification` on Android.

Reply:
One way have more parity between platforms would be to update the callback in `performVerification` to pass the merchant a `next` function that they call instead of calling `continuePerformVerification`.

Reply:
But it might also make sense to just keep `performVerification` and `continuePerformVerification` for the sake of minimizing breaking changes for merchants.

Reply:
We reverted the renaming of these methods - so they are back to `performVerification` and `continuePerformVerification`